core/storage/user/gae_models.py

```diff
@@ -72,13 +71,28 @@ class UserSettingsModel(base_models.BaseModel):
 
     @classmethod
     def is_normalized_username_taken(cls, normalized_username):
-        """Returns whether or a given normalized_username is taken."""
+        """Returns whether or not a given normalized_username is taken.
+
+        Args:
+            normalized_username: str. The given user's normalized username.
+
+        Returns:
+            True if the normalized_username is taken, False otherwise.
```

Comment:
Sorry, forgot this in previous review. Type should be specified at outset, e.g.:

bool. Whether the normalized_username has already been taken.